modules/ROOT/nav.adoc

```diff
@@ -305,13 +309,20 @@
  ** xref:web-service/web-service-consumer-reference.adoc[Web Service Consumer Connector Reference]
  ** xref:web-service/web-service-consumer-configure.adoc[Configure the Web Service Consumer]
  ** xref:web-service/web-service-consumer-consume.adoc[Consume a Web Service]
+ * xref:websocket/websocket-connector.adoc[WebSocket Connector]
+  ** xref:websocket/websocket-connector-key-concepts.adoc[Key Concepts]
+  ** xref:websocket/websocket-connector-configuration.adoc[Configuration]
+  ** xref:websocket/websocket-connector-management.adoc[Management]
+  ** xref:websocket/websocket-connector-triggers.adoc[Trigger Details]
+  ** xref:websocket/websocket-connector-operations.adoc[Operation Details]
+  ** xref:websocket/websocket-connector-cloudhub.adoc[Deploy WebSocket Connector to CloudHub]
 * xref:workday/workday-connector.adoc[Workday Connector]
- ** xref:workday/workday-connector-design-center.adoc[Workday Design Center Configuration]
- ** xref:workday/workday-connector-studio.adoc[Workday Studio Configuration]
- ** xref:workday/workday-connector-config-topics.adoc[Workday Additional Configuration]
- ** xref:workday/workday-connector-xml-maven.adoc[Workday XML Maven Configuration]
- ** xref:workday/workday-connector-examples.adoc[Workday Examples]
- ** xref:workday/workday-reference.adoc[Workday Connector Reference]
+ ** xref:workday/workday-connector-design-center.adoc[Design Center Configuration]
```

Comment:
The original documentation also contained an extensive and detailed example for a stocks application. That's missing too

Reply:
Added!